sim/dex.ts

```diff
@@ -537,7 +537,8 @@ export class ModdedDex {
 		} else if (name.startsWith('item:')) {
 			effect = this.getItem(name.slice(5));
 		} else if (name.startsWith('ability:')) {
-			effect = this.getAbility(name.slice(8));
+			const ability = this.getAbility(name.slice(8));
+			effect = Object.assign(Object.create(ability), {id: 'ability:' + ability.id});
```

Comment:
Surely this is a horrible idea? @Zarel 

Doesn't this overwrite an id field which should be type `ID`?

Reply:
luckily getEffect is unused to call abilities and this only lasts for a month; you can ask @urkerab for more info though.

Reply:
I strongly disagree with adding bugs, even temporarily, for niche OM usecases.